packages/playground/data-liberation/src/WP_Directory_Reader.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class WP_Serialized_Pages_Reader {
```

Comment:
Should this class be named `WP_Directory_Reader` to match the file name?

Reply:
Good spot! I don't like either name TBH but I don't have any better one yet